package.json

```diff
@@ -15,8 +15,8 @@
   "scripts": {
     "cm": "npx git-cz",
     "dev": "docz dev",
-    "test": "cross-env CI=1 react-scripts test --env=jsdom",
-    "test:watch": "react-scripts test --env=jsdom",
+    "test": "cross-env CI=1 react-scripts test --env=jsdom --transformIgnorePatterns \"node_modules/(?!react-dnd)/\"",
```

Comment:
I had to insert this new flag because after upgrading the `react-dnd` version, an issue came up while running tests. It was necessary to add this param in the test script for them to pass.

For further information, please check the following link from `react-dnd` repo:
https://github.com/react-dnd/react-dnd/issues/1396